book.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'relative'
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent_book(rental)
+    rentals << rental
+  end
```

Comment:
- [ ] In your implementation, the association is not being created since all you are doing is pushing the rental into the `@rentals` array. In order for this association to be created we need to create a new rental by calling the rental class. Example:

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```